library/java/org/chromium/net/ChromiumNetManifest.xml

```diff
@@ -2,7 +2,7 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
           package="org.chromium.net">
     <uses-sdk
-            android:minSdkVersion="9"
+            android:minSdkVersion="21"
```

Comment:
What's the impact of moving this from 9 to 21? Does this prevent it from running on older Androids?

Reply:
There's no impact since the Envoy-mobile [engine](https://github.com/envoyproxy/envoy-mobile/blob/main/library/java/io/envoyproxy/envoymobile/engine/AndroidEngineManifest.xml) requires 21+.
Cronvoy won't be able to support 9-21 anyway.

Reply:
Ah, good point!